ted_sws/core/model/metadata.py

```diff
@@ -81,6 +81,7 @@ class NormalisedMetadata(Metadata):
     eforms_subtype: str
     xsd_version: str
     published_in_cellar_counter: int = Field(default=0)
+    is_eForm: Optional[bool]
```

Comment:
- Keep snake style in code, rename is_eForm to is_eform.
- Provide default value as False.

Reply:
replaced name and added default value